Source/Core/Core/Core.cpp

```diff
@@ -62,6 +62,15 @@
 #include "VideoCommon/OnScreenDisplay.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+//This can be removed when we move to VS2015
```

Comment:
"when we move to VS2015 and Android stops pretending to support things that don't work."